.github/CODEOWNERS

```diff
@@ -1 +1,2 @@
-* @adamdecaf
\ No newline at end of file
+* @adamdecaf
+* @vxio
```

Comment:
I've never seen it like this, but not sure if it's wrong.  I've always seen it all on one line, e.g.
```
* @adamdecaf @vxio 
```